var/spack/repos/builtin/packages/jacamar-ci/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import re
+
+from spack.package import *
+
+
+class JacamarCi(Package):
+    """Jacamar CI is a HPC focused CI/CD driver for the GitLab custom executor."""
+
+    homepage = "https://gitlab.com/ecp-ci/jacamar-ci"
+    url = "https://gitlab.com/ecp-ci/jacamar-ci/-/archive/v0.24.0/jacamar-ci-v0.24.0.tar.gz"
+    git = "https://gitlab.com/ecp-ci/jacamar-ci.git"
+
+    maintainers("paulbry")
+
+    license("Apache-2.0 OR MIT")
+
+    version("develop", branch="develop")
+    version("0.24.1", sha256="fe1036fee2e97e38457212bf1246895803eeb6e1a6aa1ecd24eba1d3ea994029")
+    version("0.23.0", sha256="796679e13ece5f88dd7d4a4f40a27a87a6f3273085bb07043b258a612a4b43d3")
+
+    conflicts("platform=darwin", msg="Jacamar CI does not support MacOS")
+
+    depends_on("go@1.23.0:", type="build", when="@0.24.0:")
```

Comment:
I'm never sure with go packages, but https://gitlab.com/ecp-ci/jacamar-ci/-/blob/v0.24.1/go.mod?ref_type=tags#L3 indicates 1.22 for 0.24

Reply:
That is correct, I was going off the Go version used in the release but this makes more sense.